hoprd/hoprd/example_cfg.yaml

```diff
@@ -3,267 +3,213 @@ identity:
   # Path to the identity file
   # A new identity file will be created if it does not exist
   file: path/to/identity.file
-
```

Comment:
The new formatter hates human readability.

Reply:
It will keep it in place from now on.